core/client/src/test/java/alluxio/client/file/UnderFileSystemUtilsTest.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file;
+
+import alluxio.AlluxioURI;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.underfs.options.MkdirsOptions;
+import alluxio.wire.FileInfo;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.IOException;
+
+/**
+* Tests {@link UnderFileSystemUtils}.
+*/
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({FileSystem.class, UnderFileSystem.class})
+public class UnderFileSystemUtilsTest {
+
+  private FileSystem mFileSystem;
+  private UnderFileSystem mUnderFileSystem;
+
+  // Test paths in Alluxio
+  AlluxioURI mAlluxioRoot;
```

Comment:
nit: seems like all those URIs and UFS paths are static. What do you think of making these constant with full name? would the following format have slightly better readability?

```
final String mUfsRoot = "/ufs";
final String mUfsA = "/ufs/a";
final String mUfsB = "/ufs/a/b";
final String mUfsFile = "/ufs/a/b/file";
```